packages/teleport-plugin-common/src/builders/ast-builders.ts

```diff
@@ -188,7 +188,10 @@ export const appendAssetsAST = (
   bodyNode: types.JSXElement
 ) => {
   assets.forEach((asset) => {
-    const assetPath = UIDLUtils.prefixAssetsPath(options.assetsPrefix, asset.path)
```

Comment:
Please use [optional-chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) `asset?.path`. We follow the similar everywhere